src/components/Cluster/ClusterDetail/NodePool.tsx

```diff
@@ -134,13 +203,15 @@ class NodePool extends Component {
             <NodesWrapper
               style={{
                 background:
-                  current < desired ? theme.colors.goldBackground : null,
+                  current < desired ? theme.colors.goldBackground : undefined,
               }}
             >
               {current}
             </NodesWrapper>
             {/* Applying style here because is super specific for this element and can't use nth-child with emotion */}
             <NodePoolDropdownMenu
+              // This is a div, but still TS doesn't like 'style' attr...
+              // @ts-ignore
               style={{ justifySelf: 'right' }}
```

Comment:
The error is there because the component doesn't actually use that prop
https://github.com/giantswarm/happa/blob/318e6957209e571a06abb68776cbb9f6e7aad2c9/src/components/Cluster/ClusterDetail/NodePoolDropdownMenu.js